lib/util/http/auth.spec.ts

```diff
@@ -69,6 +70,45 @@ describe(getName(), () => {
       `);
     });
 
+    it('github token', () => {
+      const opts: GotOptions = {
+        headers: {},
+        token: 'XXXX',
+        hostType: PLATFORM_TYPE_GITHUB,
+      };
+
+      applyAuthorization(opts);
+
+      expect(opts).toMatchInlineSnapshot(`
+        Object {
+          "headers": Object {
+            "authorization": "token XXXX",
+          },
+          "hostType": "github",
+          "token": "XXXX",
+        }
+      `);
+    });
```

Comment:
Please use `toEqual` matcher, so this didn't get silently updated by jest snapshot updates.